common/rtc_manager_rviz_plugin/src/rtc_manager_panel.cpp

```diff
@@ -0,0 +1,314 @@
+  const size_t num_modules = module_size;
+  auto_mode_table_->setRowCount(num_modules);
+  for (size_t i = 0; i < num_modules; i++) {
+    auto * rtc_auto_mode = new RTCAutoMode();
+    rtc_auto_mode->setParent(this);
+    // module
+    {
+      rtc_auto_mode->module_name = getModuleName(static_cast<uint8_t>(i));
+      std::string module_name = rtc_auto_mode->module_name;
+      auto label = new QLabel(QString::fromStdString(module_name));
+      label->setAlignment(Qt::AlignCenter);
+      label->setText(QString::fromStdString(module_name));
+      auto_mode_table_->setCellWidget(i, 0, label);
+    }
+    // mode button
+    {
+      rtc_auto_mode->auto_module_button_ptr = new QPushButton("auto mode");
+      connect(
+        rtc_auto_mode->auto_module_button_ptr, &QPushButton::clicked, rtc_auto_mode,
+        &RTCAutoMode::onChangeToAutoMode);
+      auto_mode_table_->setCellWidget(i, 1, rtc_auto_mode->auto_module_button_ptr);
+      rtc_auto_mode->manual_module_button_ptr = new QPushButton("manual mode");
+      connect(
+        rtc_auto_mode->manual_module_button_ptr, &QPushButton::clicked, rtc_auto_mode,
+        &RTCAutoMode::onChangeToManualMode);
+      auto_mode_table_->setCellWidget(i, 2, rtc_auto_mode->manual_module_button_ptr);
+    }
+    // current mode
+    {
+      QString mode = QString::fromStdString("INIT");
+      rtc_auto_mode->auto_manual_mode_label = new QLabel(mode);
+      rtc_auto_mode->auto_manual_mode_label->setAlignment(Qt::AlignCenter);
+      rtc_auto_mode->auto_manual_mode_label->setText(mode);
+      auto_mode_table_->setCellWidget(i, 3, rtc_auto_mode->auto_manual_mode_label);
+    }
+    auto_modes_.emplace_back(rtc_auto_mode);
+  }
+  auto_mode_table_layout->addWidget(auto_mode_table_);
+
+  v_layout->addLayout(auto_mode_table_layout);
+
+  // statuses
+  auto * rtc_table_layout = new QHBoxLayout;
+  {
+    auto vertical_header = new QHeaderView(Qt::Vertical);
+    vertical_header->hide();
+    auto horizontal_header = new QHeaderView(Qt::Horizontal);
+    horizontal_header->setSectionResizeMode(QHeaderView::Stretch);
+    rtc_table_ = new QTableWidget();
+    rtc_table_->setColumnCount(7);
+    rtc_table_->setHorizontalHeaderLabels(
+      {"ID", "Module", "AW Safe", "OP Safe", "AutMode", "StartDistance", "FinishDistance"});
+    rtc_table_->setVerticalHeader(vertical_header);
+    rtc_table_->setHorizontalHeader(horizontal_header);
+    rtc_table_layout->addWidget(rtc_table_);
+    v_layout->addLayout(rtc_table_layout);
+  }
+  setLayout(v_layout);
+}
+
+void RTCManagerPanel::onInitialize()
+{
+  raw_node_ = this->getDisplayContext()->getRosNodeAbstraction().lock()->get_raw_node();
+
+  sub_rtc_status_ = raw_node_->create_subscription<CooperateStatusArray>(
+    "/api/external/get/rtc_status", 1, std::bind(&RTCManagerPanel::onRTCStatus, this, _1));
+
+  client_rtc_commands_ = raw_node_->create_client<CooperateCommands>(
+    "/api/external/set/rtc_commands", rmw_qos_profile_services_default);
```

Comment:
It seems `client_rtc_commands_` is not used.
Could you add the feature to send RTC command?

Reply:
@rej55 
I thought to add this feature in this PR but it seems difficult to send rtc_commands in a good way because rtc commands changes during aw is runnning.
So I want this feature to add in another PR.